### PR TITLE
Updates to the Currency Modal implementation.

### PR DESCRIPTION
**Pull request by mattmaloof**
Added front-end and back-end to the currency modal.

image
image

#JIRA LINK

### CODE REVIEW

File: DeBlockWallet/ViewController.swift
Comment (gisele-prog) [inline DeBlockWallet/ViewController.swift:1]: looks good to me

File: DeBlockWallet/CurrencySelectionViewController.swift
Comment (gisele-prog) [inline DeBlockWallet/CurrencySelectionViewController.swift:1]: loks good to me

**Comment by ChopperTheKing** [comment]
I like organized it looks.

**Review by ChopperTheKing — COMMENTED** [review]
Excellent organization

File: DeBlockWallet/ViewController.swift [hunk 1]
```diff
@@ -11,8 +11,34 @@ class ViewController: UIViewController {
...
+        let currencySelectionVC = CurrencySelectionViewController()
+        
+        currencySelectionVC.modalPresentationStyle = .formSheet
+        
+        currencySelectionVC.completion = { [weak self] currency in print("Selected currency is \(currency)")
+            
+        }
+        
+        present(currencySelectionVC, animated: true, completion: nil)
+    }
```
Comment (k-code00) [inline DeBlockWallet/ViewController.swift:41]: lets try to compress code next time -> could have been 20 lines of code instead of 40

**Review by k-code00 — COMMENTED** [review]
great job with the UI

**Pull request merged**